service/ziti-tunnel/cmd/identities.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
```

Comment:
"NAME HERE" and "<EMAIL ADDRESS>" :) most of the files have the header to copy. Hopefully those don't have "<EMAIL ADDRESS>"?